.travis.yml

```diff
@@ -4,6 +4,8 @@ php:
   - 5.5
   - 5.6
 
+sudo: true
```

Comment:
The solr install script requires sudo, we can:
- adapt adapt the install script
- install solr in a different way

but like this, we test the install script as every user would use it, and skipping the test is not an option.